Core/include/Acts/Visualization/GeometryVisualization.hpp

```diff
@@ -0,0 +1,42 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Polyhedron.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Visualization/IVisualization.hpp"
+
+namespace Acts {
+
+namespace Visualization {
+
+/// Helper method to Surface objects
+///
+/// @param helper [in, out] The visualization helper
+/// @param surface The surface to be drawn
+/// @param gctx The geometry context for which it is drawn
+/// @param transform An option additional transform
+/// @param lseg The number of segments for a full arch (if needed)
+/// @param triangulate The (optional) boolean diretive to triangulate the faces
+/// @param color the (optional) color of the object to be written
+static inline void drawSurface(
+    IVisualization& helper, const Surface& surface, const GeometryContext& gctx,
+    const Transform3D& transform = Transform3D::Identity(), size_t lseg = 72,
+    bool triangulate = false,
+    const IVisualization::ColorType& color = {120, 120, 120}) {
+  // Drawing the polyhedron representation of surfaces
+  Polyhedron phedron = surface.polyhedronRepresentation(gctx, lseg);
+  phedron.move(transform);
+  phedron.draw(helper, triangulate, color);
+}
```

Comment:
Provocative question, do we need this extra function then?

Reply:
If you keep it, can you just make it a static function on `IVisualization`? Then we don't need this extra file and namespace.

Reply:
It will make it consistent throughout - what you see is part one of the entire package, I will add `drawVolume()` which is still very simple, but if you come to `drawTrackingVolume()` or `drawBoundarySurface()` where you add normal vectors, or last `drawBoundParameters()` where you draw error ellipse and error cone, there's gonna be a lot of code which I don't want to get into `IVisualization`, but prepares our EDM and geometry to use `IVisualization`.

The surfaces are just as lucky as they are that they are so simple.

Reply:
Ok